src/applications/mhv-medications/containers/RxBreadcrumbs.jsx

```diff
@@ -23,32 +31,59 @@ const RxBreadcrumbs = () => {
     [location, prescription, pagination?.currentPage],
   );
 
-  return (
-    <div className="no-print">
-      {breadcrumbs.length > 0 &&
-        !location.pathname.includes(medicationsUrls.subdirectories.DETAILS) && (
-          <VaBreadcrumbs
-            uswds
-            wrapping
-            label="Breadcrumb"
-            data-testid="rx-breadcrumb"
-            breadcrumbList={breadcrumbs}
-            className="no-print va-breadcrumbs-li vads-u-margin-bottom--neg1p5 vads-u-display--block"
-          />
-        )}
-      {location.pathname.includes(medicationsUrls.subdirectories.DETAILS) && (
-        <div className="include-back-arrow vads-u-margin-bottom--neg1p5 vads-u-padding-y--3">
-          <va-link
-            href={`${
-              medicationsUrls.MEDICATIONS_URL
-            }?page=${pagination?.currentPage || 1}`}
-            text="Back to medications"
-            data-testid="rx-breadcrumb-link"
-          />
-        </div>
-      )}
-    </div>
-  );
+  let content = null;
+
+  if (
+    !isDisplayingDocumentation &&
+    location.pathname.includes(medicationsUrls.subdirectories.DOCUMENTATION)
+  ) {
+    return null;
+  }
```

Comment:
@vmenshutin-bylight if you're talking about the line above, I'm not sure I understand. This will be going away as soon as the feature flag is removed and refill doesn't use the FF anymore. Am I missing something?